File: Java Solutions/two_sum.java
import java.util.Arrays;

public class two_sum {

	public int[] twoSum(int[] nums, int target) {
        int arrayLength = nums.length;
        int counter = 0;
        int[] resultArray = new int[0]; 
        
        while(counter < arrayLength){
            int pickedNum = nums[counter];
            int pickedIndex = counter;
            int otherCounter = 0;
            while(otherCounter < arrayLength){
                
                if(pickedNum == nums[otherCounter]  && counter==otherCounter) {
					break;
				}
				
                
                int total = pickedNum + nums[otherCounter];
                if(total == target){
                    resultArray = new int[2];
                    resultArray[0] = pickedIndex;
                    resultArray[1] = otherCounter;
                    break;
                }
                otherCounter++;
            }
            
            if(resultArray.length == 2){
                break;
            }
            
            counter++;
        }
        
        
        return resultArray; 
    }
	
}

File: twoSum.py
class Solution(object):
    def twoSum(self, nums, target):
       indices = []
       counter = 0
       solFound = False

       for element in nums:
           remain = target - element
           
           otherCounter = 0
           for item in nums:
               if remain == item and otherCounter != counter:
                   indices.append(counter)
                   indices.append(otherCounter)
                   solFound = True
               otherCounter+=1

           if solFound:
               break

           counter+=1

       return indices
